app/controllers/work_packages/date_picker_controller.rb

```diff
@@ -157,7 +157,29 @@ def datepicker_modal_component
                                                          focused_field:,
                                                          triggering_field: params[:triggering_field],
                                                          touched_field_map:,
-                                                         date_mode:)
+                                                         date_mode:,
+                                                         live_region_message: live_region_message)
+  end
+
+  def live_region_message
+    I18n.t(
+      "work_packages.datepicker_modal.update_inputs_aria_live_message",
+      start_date: @work_package.start_date,
+      due_date: @work_package.due_date,
+      duration: @work_package.duration,
+      working_days_only: working_days_label,
+      scheduling: scheduling_label
+    )
+  end
+
+  def working_days_label
+    key = @work_package.ignore_non_working_days ? "true" : "false"
+    I18n.t("activerecord.attributes.work_package.include_non_working_days.#{key}")
```

Comment:
You can skip the intermediate variable and just use interpolation:

```suggestion
    I18n.t("activerecord.attributes.work_package.include_non_working_days.#{@work_package.ignore_non_working_days}")
```